app/uk/gov/hmrc/agentclientrelationshipsfrontend/models/TrackInformationSorted.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentclientrelationshipsfrontend.models
+
+import uk.gov.hmrc.agentmtdidentifiers.model.Service
+
+import java.time.LocalDateTime
+
+case class TrackInformationSorted(
+  clientType: Option[String],
+  service: Option[Service],
+  clientId: String,
+  clientIdType: String,
+  clientName: Option[String],
+  status: String,
```

Comment:
Should it be some sort of enum ?

Reply:
As part of this ticket I wasn't going to refactor the 2 models I had to bring in to satisfy the as-is endpoints - they are brought in from agents-invitations-frontend as-is to allow us to all start working on stuff like this - these models are only used to satisfy parameters in routes and the contents are not used yet - which, of course, could mean it's the ideal time to change the types before they are used in anger! I would like to talk to the team about whether this ticket scope can cover this.